packages/vscode-extension/src/handlers.ts

```diff
@@ -759,11 +693,11 @@ export async function cmdHdlLoadTreeView(context: ExtensionContext) {
     try {
       switch (node.contextValue) {
         case "signedinM365": {
-          signOutM365();
+          signOutM365(true);
```

Comment:
we do sign in but the function name here is "signOutXXX()", do we do it on purpose?